src/app/users/[username]/page.module.scss

```diff
@@ -0,0 +1,174 @@
+@use '@/styles/ohma';
+
+.pageWrapper {
+    background: ohma.$colors-white;
+    box-shadow: 0 3px 6px rgba(0,0,0,.16),0 3px 6px rgba(0,0,0,.23);
+    border-radius: 1em;
+    margin: 2em;
+}
+
+.standard {
+    background-color: ohma.$colors-secondary;
+}
+
+.bronze {
+    background: linear-gradient(to bottom right, #673208, #db6c2b);
+}
+
+.silver {
+    background: linear-gradient(to bottom right, #ededed, #bdbdbd);
+}
+
+.gold {
+    background: linear-gradient(to bottom right, #B69312, #FFDF6F);
+}
+
+.diamond {
```

Comment:
I might be blind, but are these classes currently used somewhere? I'm guessing they're for the "kappemann" flairs?

Reply:
Never mind, saw your comment further down.